### DISCUSSION TITLE
I approved for the student pack, but can not use the feaures!

### DISCUSSION THREAD

**Discussion by Zada5** [community/community#125837] [GitHub Education]
### Hi everyone,

I am a university student. 
I submitted a request for a GitHub education pack, And my request was approved. I can see this in the mail and on the GitHub website!
I tried to connect to GitHub co-pilot - but it is not working! every time I tried there was the message:  "I do not have a co-pilot account." 
I looked in the QNA, and there was no answer to that. can anyone help? 
Thanks

**Comment by davevad93** [comment]
Hi @Zada5 ,

See my answer [here](https://github.com/orgs/community/discussions/125838#discussioncomment-9563072), maybe it can help you.
